CONTRIBUTING.md

```diff
@@ -24,17 +24,35 @@ If this command throws an error run it as a `sudo`:
 sudo npm run patch:hosts
 ```
 
-1. [This repo] etc/hosts setup is required to access the application in the browser.
-2. Clone `chrome-service-backend` and checkout [this branch](https://github.com/andywaltlova/chrome-service-backend/tree/feat/digital-roadmap-nav)
-   * the changes on that branch are required to see the application in navigation menu
-   * run `make dev-static` to start the service
-3. [This repo] ```npm install```
-4. [This repo] ```npm run start```
-   * set the environment variable `CHROME_SERVICE` to the port that it is listening on (by default `8000`). For example, `CHROME_SERVICE=8000 npm run start`.
-5. [This repo] Select environment to run the app in, for dev you can use e.g. `stage-stable`, url for that would be `https://stage.foo.redhat.com:1337/insights/digital-roadmap`
-6. [This repo] Open browser in URL listed in the terminal output
-   * If you see cert error continue to the page anyway (this is expected), I had no problem with firefox but chrome may be more strict
-   * For the credentials you can check our shared password manager, if you don't have required rights at your own account
+### Start the Chrome services backend ###
+
+1. Clone the fork of `chrome-service-backend` with Digital Roadmap added to the navigation and start the service.
+
+   ```
+   git clone https://github.com/samdoran/chrome-service-backend.git --origin samdoran --branch add-digital-roadmap-nav
+   cd chrome-service-backend
+   ```
+
+1. Run `make dev-static`.
+
+###
+
+In the checkout of this repository, do the following:
+
+1. Install dependencies.
+
+   ```npm install```
+
+1. Start the application. Set `CHROME_SERVICE=[port number]` if the Chrome serivece is listening on a port other than the default of 8000.
```

Comment:
Cool! I was thinking I have to set the env var always, didn't know that 8000 is default. Works without setting the var, great :)